pkgs/applications/networking/calls/default.nix

```diff
@@ -94,6 +94,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "A phone dialer and call handler";
+    longDescription = "GNOME Calls is a phone dialer and call handler. `programs.calls` is recommended.";
```

Comment:
Maybe try to be more descriptive that it is a NixOS option and it needs to be set to true.

Reply:
Done.